themes/atomic.omp.json

```diff
@@ -0,0 +1,326 @@
+        {
+          "type": "rust",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#925837",
+          "background": "#f3f0ec",
+          "properties": {
+            "prefix": "\uE7a8 ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "dart",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#055b9c",
+          "background": "#e1e8e9",
+          "properties": {
+            "prefix": "\uE798 ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "angular",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#ce092f",
+          "background": "#ffffff",
+          "properties": {
+            "prefix": "\uE753 ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "julia",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#359a25",
+          "background": "#945bb3",
+          "properties": {
+            "prefix": "<#ca3c34>\uE624</> ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "ruby",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#9c1006",
+          "background": "#ffffff",
+          "properties": {
+            "prefix": "\uE791 ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "azfunc",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#5398c2",
+          "background": "#ffffff",
+          "properties": {
+            "prefix": "\uf104<#f5bf45>\uf0e7</>\uf105 ",
+            "postfix": ""
+          }
+        },
+        {
+          "type": "aws",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#faa029",
+          "background": "#565656",
+          "properties": {
+            "prefix": "\uE7AD ",
+            "postfix": "",
+            "template": "{{.Profile}}{{if .Region}}@{{.Region}}{{end}}"
+          }
+        },
+        {
+          "type": "kubectl",
+          "style": "diamond",
+          "leading_diamond": " \uE0B6",
+          "trailing_diamond": "\uE0B4",
+          "foreground": "#ffffff",
+          "background": "#316ce4",
+          "properties": {
+            "prefix": "\uFD31 ",
+            "postfix": "",
+            "template": "{{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}}"
+          }
+        },
+        {
+          "type": "os",
+          "style": "diamond",
+          "leading_diamond": "\uE0B6",
```

Comment:
I would add `"leading_diamond": " \uE0B6",` as without it, you get the previous language segment attached to this one while every other one has a space separation. But other than that 👍🏻

Reply:
done